cmake/BuildLibCURL.cmake

```diff
@@ -189,6 +200,8 @@ macro(curl_build)
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_GSSAPI=OFF")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_LIBSSH=OFF")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_LIBPSL=OFF")
+    list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_USE_LIBUV=OFF")
+    list(APPEND LIBCURL_CMAKE_FLAGS "-DUSE_NGHTTP2=ON")
```

Comment:
It is already here on the line 80.

Reply:
Thsnks, removed USE_NGHTTP2 option, but didn't find CURL_USE_LIBUV do you mean it also appears elsewhere?

Reply:
Nope, this comment was about the nghttp2 flag. Thanks!